test_scripts/Defects/8_1/3831_SDL_calculates_playback_time_based_on_data_provided_by_HMI.lua

```diff
@@ -0,0 +1,109 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/3831
+---------------------------------------------------------------------------------------------------
+-- Description: Check SDL calculates playback time based on 'samplingRate' and 'bitsPerSample'
+-- provided by HMI
+--
+-- Steps:
+-- 1. HMI provides some values for 'samplingRate' and 'bitsPerSample' within 'UI.GetCapabilities' response
+-- 2. Navi App registers and starts streaming Audio
+-- SDL does:
+--  - send 'Navigation.OnAudioDataStreaming(true)' to HMI once streaming is started
+--  - start internal timer based on data provided by HMI according to formula:
+--    timeout = 1000 * data_size / (sampling_rate * bits_per_sample / 8) + latency
+--  - send 'Navigation.OnAudioDataStreaming(true)' to HMI once timer expires
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('user_modules/sequences/actions')
+local runner = require('user_modules/script_runner')
+local utils = require("user_modules/utils")
+local color = require("user_modules/consts").color
+local hmi_values = require('user_modules/hmi_values')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local testCases = {
+  [01] = { samplingRate = 16000, bitsPerSample = 16 },
+  [02] = { samplingRate = 16000, bitsPerSample = 8 },
+  [03] = { samplingRate = 22000, bitsPerSample = 8 }
```

Comment:
Should we extend coverage for all available combinations with parameters according to API?
For ex:   
  [04] = { samplingRate = 22000, bitsPerSample = 16 },
  [05] = { samplingRate = 8000, bitsPerSample = 8 },
  [06] = { samplingRate = 8000, bitsPerSample = 16},
  [07] = { samplingRate = 44000, bitsPerSample = 8 },
  [08] = { samplingRate = 44000, bitsPerSample = 16 }

Reply:
Anyway additional test cased added in 13e162f